ui/app/src/components/AuditGrid/AuditGridUI.tsx

```diff
@@ -371,7 +371,7 @@ export function AuditGridUI(props: AuditGridUIProps) {
         sortingOrder={['desc', 'asc']}
         sortModel={sortModel}
         sortingMode="server"
-        autoHeight
+        autoHeight={Boolean(auditLogs.length)}
```

Comment:
There was a visual issue when there were no logs (and Empty State shows) and the autoHeight was true. When there are rows the autoHeight works properly.